packages/gatsby-source-shopify/README.md

```diff
@@ -76,29 +77,32 @@ module.exports = {
     {
       resolve: "gatsby-source-shopify",
       options: {
-        password: process.env.SHOPIFY_SHOP_PASSWORD,
-        storeUrl: process.env.GATSBY_SHOPIFY_STORE_URL,
+        password: process.env.SHOPIFY_APP_PASSWORD,
+        storeUrl: process.env.GATSBY_MYSHOPIFY_URL,
+        salesChannel: process.env.SHOPIFY_APP_ID, // Optional but recommended
```

Comment:
I don't remember seeing this in the code - I'm curious to know, what exactly does it do?

Reply:
password and storeUrl are required in order to connect to the Shopify Admin API, but sales channel is an optional variable that allows us to source specific products, collections, etc. based on whether they have been published to that sales channel or not. For example, if I had a store and wanted to publish some products to the website, but make others only available for purchase off of another platform like Amazon, this would be quite useful. You can alternatively set the GATSBY_SHOPIFY_SALES_CHANNEL variable like the README states, and this is what Gatbsy Cloud does with the Shopify integration.